app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:id])
+    @like = @post.likes.build
+    @like.author = current_user
+
+    redirect_to user_posts_path(current_user) if @like.save
+    render plain: 'Error occured while saving data'
```

Comment:
- [ ] Nice working on your likes controller you need remove `render plain: 'Error occured while saving data'` its currently throwing an error when you try to like a post